controllers/api/index.js

```diff
@@ -3,7 +3,8 @@ const userRoutes = require("./userRoutes");
 const questionRoutes = require('./questionRoutes');
 // const commentRoutes = require('./commentRoutes');
 
-router.use("/users", userRoutes);
```

Comment:
no need to remove this, makes more sense to have all the user api activity under the users portion of the url